corehq/apps/analytics/templates/analytics/initial/gtm.html

```diff
@@ -5,8 +5,8 @@
   {% initial_analytics_data 'gtm.userIsDimagi' request.couch_user.is_dimagi %}
   {% initial_analytics_data 'gtm.userIsCommCareUser' request.couch_user.is_commcare_user %}
 {% endif %}
-{% if domain %}
-  {% initial_analytics_data 'gtm.domain' domain %}
+{% if domain or request.domain %}
+  {% initial_analytics_data 'gtm.domain' domain|default:request.domain %}
```

Comment:
This breaks the raw_doc page, where `domain` is the string `"Unknown"` (see [here](https://github.com/dimagi/commcare-hq/blob/4385ccd1b66464b8c70951634d0b1885317d94f5/corehq/apps/hqwebapp/doc_lookup.py#L166-L171)) and the request doesn't have a `domain` attribute. 

Sample error message: `Failed lookup for key [domain] in <WSGIRequest: GET '/hq/admin/raw_doc/?id=c9cbe65cc318b403ea60ebafe6000826'>` I don't know why django is accessing `request.domain` at all, since `domain` is present, but maybe the expression passed to `default` gets evaluated regardless of whether or not it's ultimately used.

Reply:
Thanks for this. It does appear that the `default` filter evaluates the expressions regardless of whether or not it is used. This page does serve as the exception, where we'd want to use the `domain` attribute rather than `request.domain`, so I think we're stuck relying on an if/else block.